pkg/object/objmetadata_test.go

```diff
@@ -102,6 +102,30 @@ func TestParseObjMetadata(t *testing.T) {
 			inventory: &ObjMetadata{},
 			isError:   true,
 		},
+		"RBAC with underscores": {
```

Comment:
TestParseObjMetadata only tests parsing, unfortunately, not round-trip format+parse. I think this change would break the round-trip.